build/Makefile

```diff
@@ -443,42 +446,41 @@ setup-prometheus: PVC ?= true
 setup-prometheus: PV_SIZE ?= 64Gi
 setup-prometheus: SCRAPE_INTERVAL=30s
 setup-prometheus:
-	$(DOCKER_RUN) \
-		helm upgrade --install --wait prom stable/prometheus --namespace metrics \
+	$(DOCKER_RUN) helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+	$(DOCKER_RUN) helm repo update
+	$(DOCKER_RUN) helm upgrade prom stable/prometheus \
+		--install --wait --namespace metrics --create-namespace \
```

Comment:
nit: I don't see a reason for a line break before `--set` any longer and then all flags will be on one line.

Reply:
That --set line is quite long:
https://github.com/googleforgames/agones/blob/0fce1b368cedd5314e823016e687d1afe9aa8b2f/build/Makefile#L453, at 135 chars.

That beings said , `$(HELM_ARGS) -f $(mount_path)/build/prometheus.yaml` could definitely be pulled into one line, so doing that now.

Reply:
Oh sorry, I didn't notice the line break in the code review tool and thought it was a super short line instead of a rather long one. :)